client/graphite/paths/cache.go

```diff
@@ -11,6 +11,7 @@ var (
 	pathsCacheEnabled = false
 )
 
+// InitPathsCache inits paths for the cache.
```

Comment:
/paths for the cache/cache for the paths/

Reply:
Done.